utils/gen-static-stdlib-link-args

```diff
@@ -66,6 +66,11 @@ function write_linkfile {
 -latomic
 -lswiftImageInspectionShared
 $ICU_LIBS
+-ldispatch
+-lswiftDispatch
+-lDispatchStubs
+-lbsd
```

Comment:
There is no reason to assume libbsd - you can build with the stubs and without libbsd.

This file is really horrible.  If this is meant to be a valid way to build, I think that we should hardcode this into the driver.  CC @jrose-apple

Reply:
If you dont have `-lbsd` you get the following link error:

```
/home/spse/swift-build-master-test/usr/lib/swift_static/linux/libdispatch.a(init.c.o):init.c:function _dispatch_object_missing_debug: error: undefined reference to 'strlcpy'
/home/spse/swift-build-master-test/usr/lib/swift_static/linux/libdispatch.a(init.c.o):init.c:function _dispatch_debugv: error: undefined reference to 'strlcpy'
/home/spse/swift-build-master-test/usr/lib/swift_static/linux/libdispatch.a(init.c.o):init.c:function _dispatch_logv_init: error: undefined reference to 'getprogname'
```

The idea of this file is that the `static-stdlib-link-args` file can be generated at build time depending on the platform, rather than having a complex list of files in the driver with lots of per platform `#ifdefs`.

Reply:
I ended up removing these extra libs as they broke the `swift` tests anyway, so I just added them as extra link options to the `swift-integration-tests` test instead.